flutter_yaml.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: flutter.yaml Format
+permalink: /flutter_yaml
+---
+
+# {{ page.title }}
```

Comment:
The template already has this.  You'll end up with two copies if you include it again.